src/example.c

```diff
@@ -79,12 +79,23 @@ int main(void)
     //                               "application.log");
 
     sentry_init(options);
-    send_event();
-
-    sentry_shutdown();
 
-    // normally happens before sentry_shutdown
-    startup();
+    // `make run message`
+    if( strncmp(argv[1], "--crash", 4) ) {
+        printf("SentryEventType: %s\n", argv[1]);
+        // for Sentry Message
+        send_event();
+        sentry_shutdown();
```

Comment:
_nit_: We suggest to put this as close to the end of the program as possible. I'd move it just before the `return 0;` statement.